INSTALL.rst

```diff
@@ -29,7 +29,7 @@ If you get an empty output from the previous command, run:
 
 .. code-block:: console
 
-    (cds3)$ mkdir src ; cd src
+    (cds3)$ mkdir src ; cd srcbt
```

Comment:
on purpose?

Reply:
typo, although I will review the Install in general in a different PR